ci-operator/step-registry/redhat-appstudio/rhtap-installer/cli/redhat-appstudio-rhtap-installer-cli-commands.sh

```diff
@@ -0,0 +1,109 @@
+# QUAY__DOCKERCONFIGJSON=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhtap_quay_ci_token)
+# SPRAYPROXY_SERVER_URL=$(cat /usr/local/rhtap-ci-secrets/rhtap/sprayproxy-server-url)
+# SPRAYPROXY_SERVER_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/sprayproxy-server-token)
+# GITLAB__APP__CLIENT__ID=$(cat /usr/local/rhtap-ci-secrets/rhtap/gitlab_oauth_client_id)
+# GITLAB__APP__CLIENT__SECRET=$(cat /usr/local/rhtap-ci-secrets/rhtap/gitlab_oauth_client_secret)
+# GITLAB__TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gitlab_token)
+# QUAY__API_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/quay_api_token)
+
+# TPA__GUAC__PASSWORD="guac1234" # notsecret
+# TPA__KEYCLOAK__ADMIN_PASSWORD="admin123456" # notsecret
+# TPA__MINIO__ROOT_PASSWORD="minio123456" # notsecret
+# TPA__OIDC__TESTING_MANAGER_CLIENT_SECRET="ca48053c-3b82-4650-a98d-4cace7f2d567" # notsecret
+# TPA__OIDC__TESTING_USER_CLIENT_SECRET="0e6bf990-43b4-4efb-95d7-b24f2b94a525" # notsecret
+# TPA__OIDC__WALKER_CLIENT_SECRET="5460cc91-4e20-4edd-881c-b15b169f8a79" # notsecret
+# TPA__POSTGRES__POSTGRES_PASSWORD="postgres123456" # notsecret
+# TPA__POSTGRES__TPA_PASSWORD="postgres1234" # notsecret
+OPENSHIFT_API="$(yq e '.clusters[0].cluster.server' $KUBECONFIG)"
+# TAS__SECURESIGN__FULCIO__ORG_EMAIL='rhtap-qe-ci@redhat.com'
+# TAS__SECURESIGN__FULCIO__ORG_NAME='RHTAP CI Jobs'
+# TAS__SECURESIGN__FULCIO__OIDC__URL='http://localhost:3030'
+# TAS__SECURESIGN__FULCIO__OIDC__CLIENT_ID="fake-one"
+# TAS__SECURESIGN__FULCIO__OIDC__TYPE="dex"
+# NAMESPACE=rhtap
+
+yq -i 'del(.clusters[].cluster.certificate-authority-data) | .clusters[].cluster.insecure-skip-tls-verify=true' "$KUBECONFIG"
+OPENSHIFT_PASSWORD="$(cat $KUBEADMIN_PASSWORD_FILE)"
+
+timeout --foreground 5m bash  <<- "EOF"
+    while ! oc login "$OPENSHIFT_API" -u kubeadmin -p "$OPENSHIFT_PASSWORD" --insecure-skip-tls-verify=true; do
+            sleep 20
+    done
+EOF
+
+if [ $? -ne 0 ]; then
+  echo "Timed out waiting for login"
+  exit 1
+fi
+
+# clone_repo(){
+#   if [[ "${JOB_NAME}" == *"redhat-appstudio-rhtap-installer"* ]]; then
+#     echo "[INFO]Skip cloning rhtap-installer repo..."
+#     return
+#   fi
+#   echo "[INFO]Cloning rhtap-installer repo..."
+#   git clone https://github.com/redhat-appstudio/rhtap-installer.git
+#   cd rhtap-installer
+# }
+
+# wait_for_pipeline() {
+#   if ! oc wait --for=condition=succeeded "$1" -n "$2" --timeout 300s >"$DEBUG_OUTPUT"; then
+#     echo "[ERROR] Pipeline failed to complete successful" >&2
+#     oc get pipelineruns "$1" -n "$2" >"$DEBUG_OUTPUT"
+#     exit 1
+#   fi
+# }
+
+install_rhtap(){
+  echo -e "INFO: cloning repo name '$(params.git-repo)' with revision '$(params.git-revision)' from url '$(params.git-url)'"
+  echo "git clone"
+  git clone $GIT_URL
+  echo "git checkout"
+  git checkout $GIT_REVISION
+  echo "make build"
+  make build
+  echo "install"
+  ./bin/rhtap-cli deploy --config ./config.yaml --kube-config "$KUBECONFIG" --debug
```

Comment:
```
* developerHub.ingressDomain: apps.rhtap-4-14-us-east-2-p52r7.ci.stonesoupengineering.com
Error: install failed: failed post-install: 1 error occurred:
	* timed out waiting for the condition
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"sigs.k8s.io/prow/pkg/entrypoint/run.go:84","func":"sigs.k8s.io/prow/pkg/entrypoint.Options.internalRun",
"level":"error","msg":"Error executing test process","severity":"error","time":"2024-07-03T14:54:20Z"}
error: failed to execute wrapped command: exit status 1
```

@ascerra Could you please try adding debug log-level to see what exactly not working out? 
```
$ ./bin/rhtap-cli deploy --config ./config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
```